apps/web/src/views/Swap/MMLinkPools/components/MMCommitButton.tsx

```diff
@@ -71,10 +65,11 @@ export function MMSwapCommitButton({
   const { t } = useTranslation()
   // the callback to execute the swap
 
-  const swapCalls = useSwapCallArguments(rfqTrade.trade, rfqTrade.rfq, recipient)
+  // @ts-ignore
+  const swapCalls = useSwapCallArguments(rfqTrade?.trade, rfqTrade?.rfq as RFQResponse['message'], recipient)
 
   const { callback: swapCallback, error: swapCallbackError } = useSwapCallback(
-    rfqTrade.trade,
+    rfqTrade.trade as SmartRouterTrade<TradeType>,
```

Comment:
same here if the `rfqTrade` might be undefined. Pls also update the input type of `useSwapCallback` hook

Reply:
done